src/legacy/core_plugins/metrics/public/components/splits/terms.js

```diff
@@ -147,6 +147,33 @@ const SplitByTermsUi = props => {
           </EuiFormRow>
         </EuiFlexItem>
       </EuiFlexGroup>
+
+      <EuiSpacer />
+
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem>
+          <EuiFormRow
+            id={htmlId('include')}
+            label={(<FormattedMessage
+              id="tsvb.splits.terms.include"
```

Comment:
Please add the label type to id:
```suggestion
              id="tsvb.splits.terms.includeLabel"
```

Reply:
@maryia-lapata The type has been added. Thanks!